### PR TITLE
Proof reading

### PR DESCRIPTION
**Pull request by ClaireDL**
Reviewed quickly 00 and 01.
Proofread 02

### CODE REVIEW

**Comment by ColOfAbRiX** [comment]
Many files have their permissions changed. Change back to 0644

**Comment by ColOfAbRiX** [comment]
Comments on commits are meaningless

**Comment by ClaireDL** [comment]
I have removed the files that i didn't work on from my latest commit.
Does that solve the permission issue?

**Comment by ColOfAbRiX** [comment]
You removed key files of the project!!!
And the modified files contain spurious text from the git merge command.

File: 01_different_way_of_thinking.md [hunk 1]
```diff
@@ -18,7 +18,11 @@ before you can comfortably reason and work with them. They are very different fr
 imperative programming style that the majority of programmers have used and, at least for me,
 they require to give up many of the established practices of coding.
 
+<<<<<<< HEAD
```
Comment (ColOfAbRiX) [inline 01_different_way_of_thinking.md:21]: What's this leftover?

File: 01_different_way_of_thinking.md [hunk 2]
```diff
@@ -18,7 +18,11 @@ before you can comfortably reason and work with them. They are very different fr
...
+These are also the reasons why I find FP so fun and interesting, because they forced me to adapt to a
+=======
```
Comment (ColOfAbRiX) [inline 01_different_way_of_thinking.md:23]: Same here

File: 01_different_way_of_thinking.md [hunk 3]
```diff
@@ -18,7 +18,11 @@ before you can comfortably reason and work with them. They are very different fr
...
 These are also the reason why I find FP so fun and interesting, because they forced me to adapt a
+>>>>>>> d3d029f667292db712779a3bd2548a19c33abcc4
```
Comment (ColOfAbRiX) [inline 01_different_way_of_thinking.md:25]: Same here and in other files too

**Review by ColOfAbRiX — COMMENTED** [review]
You removed key files of the project!!!

And the edited files contain spurious text from the git merge command.

.gitignore still has wrong permissions

**Comment by ColOfAbRiX** [comment]
Too many issues in this PR, please create a new one.

**Pull request closed without merging**